{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "secalert@redhat.com",
      "ID": "CVE-2025-53862"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A flaw was found in Ansible. Three API endpoints are accessible and return verbose, unauthenticated responses. This flaw allows a malicious user to access data that may contain important information."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-497"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://access.redhat.com/security/cve/CVE-2025-53862",
          "refsource": "",
          "tags": [],
          "url": "https://access.redhat.com/security/cve/CVE-2025-53862"
        },
        {
          "name": "RHBZ#2379359",
          "refsource": "",
          "tags": [],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2379359"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "ADJACENT_NETWORK",
        "availabilityImpact": "NONE",
        "baseScore": 3.5,
        "baseSeverity": "LOW",
        "confidentialityImpact": "LOW",
        "integrityImpact": "NONE",
        "privilegesRequired": "LOW",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:A/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N",
        "version": "3.1"
      },
      "exploitabilityScore": 2.1,
      "impactScore": 1.4
    }
  },
  "lastModifiedDate": "2025-07-11T13:15Z",
  "publishedDate": "2025-07-11T13:15Z"
}